src/codeflare_sdk/cluster/auth.py

```diff
@@ -106,10 +118,20 @@ def login(self) -> str:
             configuration.api_key_prefix["authorization"] = "Bearer"
             configuration.host = self.server
             configuration.api_key["authorization"] = self.token
-            if self.skip_tls == False and self.ca_cert_path == None:
+
+            if self.skip_tls == False:
```

Comment:
```suggestion
            if not self.skip_tls:
```
Nitpick for a more Pythonic style.